packages/types/src/types/models-extra/dashboard-item/charts/common.ts

```diff
@@ -64,8 +64,8 @@ export type BaseChartConfig = BaseConfig & {
   renderLegendForOneItem?: boolean;
 };
 
-export const isChartConfig = (config: BaseConfig): config is BaseChartConfig => {
-  return 'type' in config && config.type === 'chart';
+export const isChartConfig = (config?: BaseConfig): config is BaseChartConfig => {
+  return (config && 'type' in config && config.type === 'chart') ?? false;
```

Comment:
I may be missing something simple, but is this different from `return config?.type === chart`?

Reply:
It's the same, just another way of doing it!